.circleci/config.yml

```
@@ -3,6 +3,11 @@ version: 2.1
 orbs:
   node: circleci/node@5.1.0
 
+parameters:
+  prismaDeployForce:
```

La convention de casse la plus répandue pour les paramètres CircleCI (pour toutes entités de configration CircleCI en fait) est kebab case
```suggestion
  prisma-deploy-force:
```